libexec/nodenv-versions

```diff
@@ -97,13 +97,17 @@ while read -r version; do
 done < <(
   {
     shopt -s nullglob
-    for path in "$versions_dir"/*; do
+    for path in "$versions_dir"/* "$versions_dir"/lts/*; do
       if [ -d "$path" ]; then
+        if [ "$(basename "$path")" == "lts" ]; then
```

Comment:
I'd like to add an additional check (and corresponding test) here. We should allow for a node version named `lts` to still function.

There may be nodenv installations in the wild that use `lts` as an alias for "latest" or "stable". So in addition to this basename test, we should _not skip_ any lts directory that contains a `bin/` directory.

Reply:
Sure, sounds like a reasonable use case 👍🏼  Updated!